### PR TITLE
KServe Incubation Proposal 

### PR DESCRIPTION
**Pull request by yuzisun**

### CODE REVIEW

**Comment by raravena80** [comment]
Great ro see this @yuzisun! 🎉

**Review by andreyvelich — COMMENTED** [review]
Excited to see this happening!
I left a few comments @yuzisun.

**Review by terrytangyuan — COMMENTED** [review]
@yuzisun Could you check the original proposal in the Google Doc? I left some edits there a while back. It would be great to incorporate those changes and address the comments there.

File: proposals/incubation/kserve.md [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+KServe uses Github Action for its CI needs. If KServe does get accepted for incubation in CNCF, the community will work towards migrating the ownership of this infrastructure to the CNCF.
+
+https://github.com/kserve/kserve/tree/master/.github
+
+## Communication Channels
+
+The KServe community includes a variety of Slack forums for general discussion, feature requests, job postings, and sub-project, platform, or geo-specific channels. Popular KServe Slack channels are listed here 
+
+- [Kubeflow kserve channel](https://kubeflow.slack.com/archives/CH6E58LNP)
```
[the remark this answers is not in the document]
Comment (terrytangyuan) [reply proposals/incubation/kserve.md:92]: We can use this link https://github.com/kserve/community?tab=readme-ov-file#questions-and-issues

**Comment by angellk** [comment]
Hi @yuzisun - we have a new process for projects that are applying to the CNCF as well as projects that are moving levels. Please open an Issue with the [new Incubation Issue template.](https://github.com/cncf/toc/issues/new?assignees=&labels=incubation&projects=&template=template-incubation-application.md&title=%5BIncubation%5D+%24PROJECT+Incubation+Application)

This issue will be closed and your new issue will be put in the queue for a TOC member to pick up depending on availability according to domain expertise and queue order. Thank you!

**Comment by terrytangyuan** [comment]
@angellk Thank you. We will open an issue for this using the correct template.

**Comment by terrytangyuan** [comment]
See https://github.com/cncf/toc/issues/1367

**Comment by johnzheng1975** [comment]
Great to see this.

**Pull request closed without merging**